components/Cards/SchoolCard/__tests__/BackSchoolCard.test.js

```diff
@@ -10,14 +11,40 @@ const locations = [
     city: 'San Francisco',
     zip: '94111',
   },
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    zip: '94111',
+  },
 ];
 
 describe('BackSchoolCard', () => {
   it('should render with required props', () => {
     createShallowSnapshotTest(
-      <BackSchoolCard cardFlipCallback={jest.fn()} locations={locations} logoSource="logoSource">
+      <BackSchoolCard
+        cardFlipCallback={jest.fn()}
+        locations={locations}
+        logoSource="logoSource"
+        alt="alt"
+      >
```

Comment:
Idk if you were able to see this since it's on the left side of the diff viewer, but 👀 
https://github.com/OperationCode/front-end/pull/300/files/483089c0a80113786c0b08237adb6fdb9ea8f3d5..a54eed8aabc5c7c89bab51df75f175872f1a8c51#r236574017